app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        if "wife" in person.keys():
+            if person["wife"]:
+                if not hasattr(Person.people[person["name"]], ""):
+                    wife = Person.people[person["wife"]]
+                    Person.people[person["name"]].wife = wife
+        if "husband" in person.keys():
+            if person["husband"]:
```

Comment:
Similar to the 'wife' check, you can simplify this by using `person.get('husband')` instead of checking if 'husband' is in person.keys() and then checking if person['husband'] is not None.